File: exercice/exercice_6/main.ts
class Client {
  accountNumber : string;
  name: string
  constructor (accountNumber: string, name: string){
    this.accountNumber = accountNumber;
    this.name = name;
    }
}

class ProfessionalClient extends Client {
  siret : number;

  constructor (accountNumber: string, name : string,siret: number){
    super(accountNumber, name);
    this.siret = siret;
  }
}

let newClient: Client = new Client ("FR92909", "Samo");
console.log(newClient.accountNumber, newClient.name);

let newProClient: ProfessionalClient = new ProfessionalClient("ab9769", "Lionel", 86889);
console.log(newProClient.accountNumber, newProClient.name,newProClient.siret)

File: exercice/exercice_6/main.java
class Playground {
public static void main(String[ ] args){
  Client newClient = new Client("Louisette", 12537);
  ClientPro newClientPro = new ClientPro ("Momo", 878688, 1980);
  System.out.println(newClient.name + "" + newClient.accountNumber);
System.out.println(newClientPro.name + "" + newClientPro.accountNumber + "" + newClientPro.siret);

  ;
  }
}

class Client{
  String name;
  int accountNumber;

  Client(String name, int accountNumber) {
    this.name= name;
    this.accountNumber=accountNumber;
  }
}

class ClientPro extends Client{
  int siret;

  ClientPro(String name, int accountNumber,int siret){
    super(name, accountNumber);
    this.siret=siret;
  }
}
